.github/ISSUE_TEMPLATE/bug.yaml

```
@@ -0,0 +1,109 @@
+    attributes:
+      label: Steps to Reproduce
+      description: Steps to reproduce the behavior
+      value: |
+        ```python
+        # Include a minimal code example that demonstrates the issue
+        from pkynetics import model_fitting_methods
+
+        # Your code here
+        ```
+
+        1. 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)